processor/configuration/src/main/java/com/redhat/service/smartevents/processor/ProcessorCatalogServiceImpl.java

```diff
@@ -126,7 +126,7 @@ private ObjectNode readSourceJsonSchema(String id) {
     }
 
     private JsonSchema getJsonSchemaFromJsonNode(ObjectNode objectNode) {
-        JsonSchemaFactory factory = JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V4);
+        JsonSchemaFactory factory = JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V201909);
```

Comment:
OpenAPI has nothing to do with the version of Json Schema. We need V201909 because it contains some of the features we use for the additional validation https://json-schema.org/specification.html